### PR TITLE
feat: #51

### PR DESCRIPTION
**Pull request by Aky22**
- Support subscriber update method

### CODE REVIEW

File: src/modules/subscribers/subscribers.test.ts [hunk 1]
```diff
@@ -74,6 +74,35 @@ describe("Subscribers", () => {
         }
     });
 
+    it("Update a subscriber", async () => {
+        const randomInt = getRandomInt();
+
+        const params: CreateOrUpdateSubscriberParams = {
```
Comment (Mantas97) [inline src/modules/subscribers/subscribers.test.ts:80]: `UpdateSubscriberParams` has to be included and used here
```suggestion
        const params: UpdateSubscriberParams = {
```

File: src/modules/subscribers/subscribers.types.ts [hunk 2]
```diff
@@ -38,6 +39,17 @@ export interface CreateOrUpdateSubscriberParams {
...
+    fields?:	        object;
+    groups?:	        Array<string>;
+    status?:	        "active" | "unsubscribed" | "unconfirmed" | "bounced" | "junk";
+    subscribed_at?:	    string;
+    ip_address?:	    string;
+    opted_in_at?:	    string;
+    optin_ip?:	        string;
+    unsubscribed_at?:	string;
+}
+
```
Comment (Mantas97) [inline src/modules/subscribers/subscribers.types.ts:52]: Should be refactored like so:
```
export interface UpdateSubscriberParams {
    fields?:	        object;
    groups?:	        Array<string>;
    status?:	        "active" | "unsubscribed" | "unconfirmed" | "bounced" | "junk";
    subscribed_at?:	    string;
    ip_address?:	    string;
    opted_in_at?:	    string;
    optin_ip?:	        string;
    unsubscribed_at?:	string;
}

export interface CreateOrUpdateSubscriberParams extends UpdateSubscriberParams {
    email:  string;
}
```

File: examples/subscribers/update.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import dotenv from 'dotenv';
+import MailerLite from '../../dist/index.mjs';
+
+dotenv.config();
+
+const mailerlite = new MailerLite({
+  api_key: process.env.API_KEY,
+});
+
+const subscriberId = '<subscriber_id>';
```
Comment (Mantas97) [inline examples/subscribers/update.js:14]: Following the patter from other examples it should be
1. `// node examples/subscribers/update.js ID_string`

2.
```suggestion
let subscriber_id = null
if (process.argv.slice(2).length) {
    subscriber_id = process.argv[2];
}
```

3. `mailerlite.subscribers.update(subscriber_id,subscriberData)`

**Review by Mantas97 — CHANGES_REQUESTED** [review]
Nice work 👏 Few adjustments needed

File: src/modules/subscribers/subscribers.test.ts (hunk 1 above)
Comment (Mantas97) [reply src/modules/subscribers/subscribers.test.ts]: @Aky22 also please import `UpdateSubscriberParams` `from "./campaigns.types"`

Comment (Aky22) [reply src/modules/subscribers/subscribers.test.ts]: I assume you mean from `./subscribers.types` . It's done :slightly_smiling_face:

**Review by Mantas97 — APPROVED** [review]
Great 👍

**Pull request merged**